policyengine_us/parameters/gov/states/ga/tax/income/agi/subtractions.yaml

```diff
@@ -0,0 +1,46 @@
+description: List of Georgia adjusted gross income subtractions.
+values:
+  2021-01-01:
+    - us_govt_interest
+    # ga_retirement_income_exclusion
+    # Military retirement income subtraction
+    # Dependent personal exemption for unborn children with a detectable human heartbeat
+    # Social security or railroad retirement
+    # Salaries and wages
+    # Individual retirement account, Keogh, SEP and SUB-S plan withdrawals
+    # Depreciation because of differences
+    # Dependent’s unearned income
+    # Income tax refunds
+    # Income from any fund
+    # Adjustment to Federal adjusted gross income
+    # Adjustment for teachers retired from the Teacher’s Re- tirement System of Georgia
+    # employersinfoodandbeverageestab- lishments
+    # minority subcontractors
+    # Path2College 529 Plan
+    # Adjustments due to Federal tax changes
+    # Combat zone pay exclusion
+    # axpayer’s donation of all or part of a kidney, liver, pancreas, intestine, lung or bone marrow during the taxable year
+    # high deductible health plans
+    # Georgia municipal bonds designated as “Build America Bonds”
+    # Georgia municipal bonds issued by the State of Georgia
+    # Interest eliminated from federal itemized deductions
+    # disabled first responder
+    # Interest eliminated from federal itemized deductions
+    # disabled first responder
+    # firefighter, payments received
+    # For a firefighter, an amount equal to 100% of any premium paid
+    # Income received by a surviving family member
+    # certain disaster relief payments
+    # For owners of a pass-through entity or entities   
+
+metadata:
+  reference:
+    - title: 2022 Form 500 Instructions
+      href: https://houpl.org/wp-content/uploads/2023/01/2022-IT-511_Individual_Income_Tax_-Booklet-compressed.pdf#page=14
+    - title: 2021 Form 500 Instructions
+      href: https://www.zillionforms.com/2021/I2122607361.PDF
+    - title: Official Code of Georgia Annotated | CH7 Income Taxes
```

Comment:
could add the correponding subchapter number after 'CH7 Income Taxes' which the link direct to